src/main/java/ReadGZIP.java

```diff
@@ -0,0 +1,28 @@
+import exceptions.ReadGZIPFileException;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.*;
+import java.util.zip.GZIPInputStream;
+
+public class ReadGZIP {
```

Comment:
Class Name should not be an action.
Maybe you can use - GZIPReader or GZIPHandler..